teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.options;
+
+import java.util.ArrayList;
+import java.util.List;
+import picocli.CommandLine;
+import tech.pegasys.teku.config.TekuConfiguration;
+
+public class ValidatorKeysOptions {
+
+  @CommandLine.Option(
+      names = {"--validator-keys"},
+      paramLabel = "<KEY_DIR>:<PASS_DIR> | <KEY_FILE>:<PASS_FILE>",
+      description =
+          "<KEY_DIR>:<PASS_DIR> will find <KEY_DIR>/**.json, and expect to find <PASS_DIR>/**.txt.\n"
+              + "<KEY_FILE>:<PASS_FILE> will expect that the file <KEY_FILE> exists, "
+              + "and the file containing the password for it is <PASS_FILE>.\n"
+              + "The path separator is operating system dependent, and should be ';' in windows rather than ':'.",
+      split = ",",
+      arity = "1..*")
+  private List<String> validatorKeys = new ArrayList<>();
```

Comment:
This isn't enough.  We also need the options to specify external signers:
* `validators-external-signer-public-keys`
* `--validators-external-signer-url`
* `--validators-external-signer-timeout`

Reply:
May as well move the old `validators-key-files` and `validators-key-password-files` options into here too just for consistency.  They're hidden and will be removed soon but its tidier to keep all the key loading options in one place.